core/src/main/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotStatements.scala

```diff
@@ -101,8 +105,18 @@ import akka.persistence.cassandra.FutureDone
         session.setSchemaMetadataEnabled(null)
         Done
       }
-      result.failed.foreach(_ => session.setSchemaMetadataEnabled(null))
-      result
-    } else keyspace
+      result.recoverWith {
+        case e =>
+          log.warning("Failed to create snapshot keyspace and tables: {}", e)
```

Comment:
Is severe enough to be an error even perhaps?

Reply:
My thinking is that since we continue anyway it's a warning.
Probably other queries will work if the tables were created by someone else.

Reply:
Hmm, and it will always be semi-atomic because they are created in order and would either fail on the first one or not conflict. Ok, I agree.

Reply:
I don't know about that. Trying to create those things concurrently is to ask for trouble, but I think it's better that we best effort try to recover from it rather than failing the Session for all infinity. If it doesn't work other requests will later fail with other error messages, such as table doesn't exist.